src/Content/Conversation.php

```diff
@@ -408,6 +409,10 @@ public function statusEditor(array $x = [], int $notes_cid = 0, bool $popup = fa
 
 			'$compose_link_title'  => $this->l10n->t('Open Compose page'),
 			'$always_open_compose' => $this->pConfig->get($this->session->getLocalUserId(), 'frio', 'always_open_compose', false),
+
+			// Dropzone
+			'$max_imagesize' => round(\Friendica\Util\Strings::getBytesFromShorthand(DI::config()->get('system', 'maximagesize')) / 1000000,1),
```

Comment:
Code standards: Please add a space after commas.

Additionally, a local property provides access to the config
```suggestion
			'$max_imagesize' => round(\Friendica\Util\Strings::getBytesFromShorthand($this->config->get('system', 'maximagesize')) / 1000000, 1),
```